From 5974513f4cd22bfadf477896353b305925974981 Mon Sep 17 00:00:00 2001
From: Nikita Travkin <nikitos.tr@gmail.com>
Date: Wed, 3 Feb 2021 21:04:50 +0500
Subject: [PATCH 091/210] arm64: dts: qcom: msm8916-modem: Enable memshare

This service is used by GPS service in many msm8916 devices.

Signed-off-by: Nikita Travkin <nikitos.tr@gmail.com>
[extracted from common msm8916-modem commit]
Signed-off-by: Stephan Gerhold <stephan@gerhold.net>
---
 arch/arm64/boot/dts/qcom/msm8916-modem.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/msm8916-modem.dtsi b/arch/arm64/boot/dts/qcom/msm8916-modem.dtsi
index 77f829e8e6a..d1c08de8d1f 100644
--- a/arch/arm64/boot/dts/qcom/msm8916-modem.dtsi
+++ b/arch/arm64/boot/dts/qcom/msm8916-modem.dtsi
@@ -18,6 +18,14 @@ &lpass {
 	status = "reserved"; /* Controlled by QDSP6 */
 };
 
+&memshare {
+	status = "okay";
+};
+
+&memshare_gps {
+	memory-region = <&gps_mem>;
+};
+
 &mpss {
 	status = "okay";
 };
-- 
2.34.1

